module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -429,6 +436,21 @@ public function setConfig($config)
      */
     public function init()
     {
+        if (empty($this->config)) {
+            throw new ILSException('Configuration needs to be set.');
+        }
+        if ($this->config['InnReach']['enabled']) {
```

Comment:
I would make this `if ($this->config['InnReach']['enabled'] ?? false) {` to fall back to a default behavior if the configuration is entirely missing.